### DISCUSSION TITLE
Could not retrieve version/homepage information

### DISCUSSION THREAD

**Discussion by tpluscode** [DefinitelyTyped/DefinitelyTyped#56518] [_Misc]
Running `npx dts-gen` lately shows this below warning every time and the generated `index.d.ts` header is incomplete

```
Warning: Could not retrieve version/homepage information: HTTP Error 301: Moved Permanently for http://...
```

I tried reinstalling global but it's not it. What do I do?

**Comment by domdomegg** [comment]
This is a bug in dts-gen. There's an outstanding PR with a fix here: https://github.com/microsoft/dts-gen/pull/174
